cmd/config/api/api.go

```diff
@@ -172,6 +179,10 @@ func LookupConfig(kvs config.KVS) (cfg Config, err error) {
 	if err != nil {
 		return cfg, err
 	}
+	replicationWorkers, err := strconv.Atoi(env.Get(EnvAPIReplicationWorkers, kvs.Get(apiReplicationWorkers)))
+	if err != nil {
+		return cfg, err
+	}
```

Comment:
```suggestion
	}
	if replicationWorkers == 0 {
	     replicationWorkers = runtime.GOMAXPROCS(0) / 2
	}
```

Reply:
if there is only one cpu this would round down to 0. config value already defaults to half available cpus if unset.

Reply:
Correct for that never choose GOMAXPROCS in the first place, always choose 100 as default @poornas - we don't if customers have slow hardware.

Reply:
ok, removed all refs to GOMAXPROCS - replication workers defaults to 100 if unset